tests/testthat/test-5prevalence.R

```diff
@@ -246,19 +212,19 @@ test_that("getRare", {
     gp_null <- GlobalPatterns
     rownames(gp_null) <- NULL
 
-    pr1 <- getRare(GlobalPatterns, detection=0.0045, prevalence = 0.25, as.relative=TRUE)
+    pr1 <- getRare(GlobalPatterns, detection=0.0045, prevalence = 0.25)
     pr1 <- which(rownames(GlobalPatterns) %in% pr1)
-    pr2 <- getRare(gp_null, detection=0.0045, prevalence = 0.25, as.relative=TRUE)
+    pr2 <- getRare(gp_null, detection=0.0045, prevalence = 0.25)
     expect_equal(pr1, pr2)
 
     # Test lias
-    alias <- getRare(gp_null, detection=0.0045, prevalence = 0.25, as.relative=TRUE)
+    alias <- getRare(gp_null, detection=0.0045, prevalence = 0.25)
     expect_equal(pr1, alias)
 
     pr1 <- getRare(GlobalPatterns, detection=0.004, prevalence = 0.1,
-                       as.relative=TRUE, rank = "Family")
+               rank = "Family")
     pr2 <- getRare(gp_null, detection=0.004, prevalence = 0.1,
```

Comment:
Convert these to more sensible threshold. 0.0045 is not sensible detection threshold for counts

Reply:
I have restored the tests since as.relative is still available. Should the threshold be modified still?

Reply:
Is there tests that test the absolute values? If not, then there should be.

detection threshold should reflect the type of values. If there are counts (and as.relative=FALSE), then the values should be integer. If the assay.type="relabundance" or as.relative=TRUE, then detection threshold should be value 0-1